qemu/tests/cfg/s390x_cpu_model.cfg

```diff
@@ -0,0 +1,40 @@
+               - model_type_z14_z14_2_z14ZR1:
+                    cpu_models = "z14 z14.2 z14ZR1"
+                    props = {"aen": 'true', "aefsi": 'true', "mepoch": 'true', "msa8": 'true', "msa7": 'true', "msa6": 'true', "msa5": 'true', "msa4": 'true', "msa3": 'true', "msa2": 'true', "msa1": 'true', "sthyi": 'true', "edat": 'true', "ri": 'true', "edat2": 'true', "vx": 'true', "ipter": 'true', "mepochptff": 'true', "vxeh": 'true', "vxpd": 'true', "esop": 'true', "iep": 'true', "cte": 'true', "bpb": 'true', "gs": 'true', "ppa15": 'true', "zpci": 'true', "sea_esop2": 'true', "te": 'true', "cmm": 'true'}
+               - model_type_gen15a_gen15b:
+                    cpu_models = "gen15a gen15b"
+                    props = {"aen": true, "vxpdeh": 'true', "aefsi": 'true', "mepoch": 'true', "msa9": 'true', "msa8": 'true', "msa7": 'true', "msa6": 'true', "msa5": 'true', "msa4": 'true', "msa3": 'true', "msa2": 'true', "msa1": 'true', "sthyi": 'true', "edat": 'true', "ri": 'true', "deflate": 'true', "edat2": 'true', "etoken": 'true', "vx": 'true', "ipter": 'true', "mepochptff": 'true', "vxeh": 'true', "vxpd": 'true', "esop": 'true', "msa9_pckmo": 'true', "vxeh2": 'true', "iep": 'true', "cte": 'true', "gs": 'true', "ppa15": 'true', "zpci": 'true', "sea_esop2": 'true', "te": 'true', "cmm": 'true'}
+               - model_type_gen16a_gen16b:
+                    cpu_models = "gen16a gen16b"
+                    props = {"nnpa": 'true', "aen": 'true', "vxpdeh": 'true', "aefsi": 'true', "mepoch": 'true', "msa9": 'true', "msa8": 'true', "msa7": 'true', "msa6": 'true', "msa5": 'true', "msa4": 'true', "msa3": 'true', "msa2": 'true', "msa1": 'true', "sthyi": 'true', "edat": 'true', "ri": 'true', "deflate": 'true', "edat2": 'true', "etoken": 'true', "vx": 'true', "ipter": 'true', "pai": 'true', "mepochptff": 'true', "vxeh": 'true', "vxpd": 'true', "esop": 'true', "msa9_pckmo": 'true', "vxeh2": 'true', "iep": 'true', "cte": 'true', "gs": 'true', "ppa15": 'true', "zpci": 'true', "rdp": 'true', "sea_esop2": 'true', "beareh": 'true', "te": 'true', "cmm": 'true', "vxpdeh2": 'true'}
+        - boot_cpu_models:
+            type = s390x_cpu_model_boot
+            auto_cpu_model = yes
+            cpu_model_check_cmd = "query-cpu-model-expansion"
+            cpu_model_check_args = '{"type": "static", "model": {"name": "host"}}'
+            Host_RHEL.m7.:
+                boot_cpu_models = 'z196,z196.2,z114;zEC12,zEC12.2,zBC12'
+            Host_RHEL.m8.:
+                boot_cpu_models = 'z13,z13.2,z13s;z14,z14.2,z14ZR1;gen15a,gen15b'
+            Host_RHEL.m9.:
+                boot_cpu_models = 'z14,z14.2,z14ZR1;gen15a,gen15b;gen16a,gen16b'
```

Comment:
Looks like a blank line is going to be added here

Reply:
Other parts ok for me, ack

Reply:
solved